compiler/z/codegen/OMRCodeGenerator.cpp

```diff
@@ -391,7 +391,7 @@ bool OMR::Z::CodeGenerator::canTransformUnsafeCopyToArrayCopy()
 
 bool OMR::Z::CodeGenerator::supportsDirectIntegralLoadStoresFromLiteralPool()
```

Comment:
Do we need this query? I do not see any other code-gen uses this and with the z10 bump, I do not see if we need to use this. This is beyond the scope of this PR but we can clean up this query if it is not useful.

Reply:
The query is used in OpenJ9 and should get migrated there.